apps/explorer/src/ui/image/Image.tsx

```diff
@@ -71,32 +82,47 @@ function BaseImage({
         >
             {status === 'loading' ? (
                 <LoadingSpinner />
-            ) : blur && status === 'loaded' ? (
-                <div className="absolute z-20 flex h-full w-full cursor-not-allowed items-center justify-center rounded-md bg-gray-100/30 text-center text-white backdrop-blur-md">
-                    <EyeClose16 />
+            ) : (
+                <div className="relative">
+                    {isBlurred && (
+                        <div
+                            className={clsx(
+                                'absolute z-20 flex h-full w-full items-center justify-center rounded-md bg-gray-100/30 text-center text-white backdrop-blur-md',
+                                visibility === VISIBILITY.HIDE &&
+                                    'pointer-events-none cursor-not-allowed'
+                            )}
+                            onClick={() => setIsBlurred(!isBlurred)}
+                        >
+                            <EyeClose16 />
+                        </div>
+                    )}
+                    <img
+                        alt={alt}
+                        src={src}
+                        srcSet={srcSet}
+                        className={imageStyles({
+                            rounded,
+                            fit,
+                            size,
+                        })}
+                        onClick={onClick}
+                        {...imgProps}
+                    />
                 </div>
-            ) : status === 'failed' ? (
-                <NftTypeImage24 />
-            ) : null}
-            {status === 'loaded' && (
-                <img
-                    alt={alt}
-                    src={src}
-                    srcSet={srcSet}
-                    className={imageStyles({
-                        rounded,
-                        fit,
-                        size,
-                    })}
-                    onClick={onClick}
-                    {...imgProps}
-                />
             )}
+            {status === 'failed' && <NftTypeImage24 />}
```

Comment:
![image](https://github.com/MystenLabs/sui/assets/122397493/3db6a779-a3b3-42c8-99ef-77ba55b63347)
seeing an issue when images fail to load

Reply:
explorer prod: 
![image](https://github.com/MystenLabs/sui/assets/122397493/b90abb0b-1193-4797-8c26-708c4bd732c5)